ext/standard/basic_functions.stub.php

```diff
@@ -76,19 +76,19 @@ function natsort(array &$array): bool {}
 
 function natcasesort(array &$array): bool {}
 
-function asort(array &$array, int $sort_flags = SORT_REGULAR): bool {}
+function asort(array &$array, int $flags = SORT_REGULAR): bool {}
 
-function arsort(array &$array, int $sort_flags = SORT_REGULAR): bool {}
+function arsort(array &$array, int $flags = SORT_REGULAR): bool {}
 
-function sort(array &$array, int $sort_flags = SORT_REGULAR): bool {}
+function sort(array &$array, int $flags = SORT_REGULAR): bool {}
 
-function rsort(array &$array, int $sort_flags = SORT_REGULAR): bool {}
+function rsort(array &$array, int $flags = SORT_REGULAR): bool {}
 
-function usort(array &$array, callable $cmp_function): bool {}
+function usort(array &$array, callable $comparison): bool {}
```

Comment:
What about `$comparator`? Personally, I would be ok with `$callback`, too

Reply:
I wasn't sure if that was a common enough word, although it was the first thing I thought of myself.  I'd be fine with it if there are no other objections.